app/app.go

```diff
@@ -377,6 +380,15 @@ func eth2Client(ctx context.Context, bnURL string) (eth2wrap.Client, error) {
 	return eth2wrap.AdaptEth2HTTP(bnClient, maxBeaconNodeTimeout), nil
 }
 
+// timeoutByValAmount returns the maximum timeout an eth2http call will have.
+// Return a timeout of (valAmount/50)*20, where 20 are the seconds to wait.
+func timeoutByValAmount(valAmount uint64) time.Duration {
```

Comment:
So 100 validators we will timeout after 40s. Useful in debugging but it should fail fast. Can we make this timeout configurable with a CLI flag instead?

Reply:
> Useful in debugging but it should fail fast.

Consider that this tool is designed to fetch this data, process and then stop to never be seen again.

Is adding a CLI toggle better than just make it work and forget about it?

Reply:
Fair enough. We can have any timeout as there are no strict timing requirements here

Reply:
Exactly. 

If this was Charon, we could never allow such timeouts.